CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -0,0 +1,259 @@
+local ICON_SIZE = 48
+local ICON_SPACING = 52
+local PIXELS_PER_STUD = 64
+
+local HOPPERBIN_ANGLE = math.rad(-45)
+local HOPPERBIN_ROTATION = CFrame.Angles(HOPPERBIN_ANGLE, 0, 0)
+local HOPPERBIN_OFFSET = Vector3.new(0, 0, -5)
+
+local Tools = {}
+local ToolsList = {}
+
+local BackpackScript = {}
+local topbarEnabled = false
+
+local player = game.Players.LocalPlayer
+
+local hopperbinPart = Instance.new("Part", workspace.CurrentCamera)
+hopperbinPart.Transparency = 1
+hopperbinPart.CanCollide = false
+hopperbinPart.Anchored = true
+hopperbinPart.Name = "GUI"
+local hopperbinGUI = Instance.new("SurfaceGui", player:WaitForChild("PlayerGui"))
```

Comment:
Now that we are a core-script we should be adding out guis to `CoreGui` instead of `PlayerGui`